package.json

```diff
@@ -8,6 +8,8 @@
   "license": "GPLv3",
   "devDependencies": {
     "bower": "^1.3.1",
+    "devtron": "^1.4.0",
+    "electron-debug": "~1.0.1",
```

Comment:
I don't remember adding them so I guess it must have been something I was experimenting with that made it's way into the patch.